x-pack/plugins/security_solution/cypress/integration/data_sources/sourcerer.spec.ts

```diff
@@ -52,9 +52,10 @@ describe('Sourcerer', () => {
     before(() => {
       createUsersAndRoles(usersToCreate, rolesToCreate);
     });
-    it(`role(s) ${secReadCasesAllUser.roles.join()} shows error when user does not have permissions`, () => {
+    it(`reader role is able to read`, () => {
```

Comment:
i think we need to revert this after Friday's changes, and add a test that a reader is able to read _after_ a writer has visited the app